### ISSUE TITLE
fix: increase text contrast for better accessibility and legibility on licensing screen

### ISSUE THREAD

**Issue by mathetos** [impress-org/give#4127]
# Bug Report

## User Story

As a Give admin, I want to be able to easily read all the settings and descriptions in the Admin without straining my eyes. 

## Current Behavior

The current Licensing UI improvements for version 2.5 have a lot of text with very low contrast making it challenging to read easily. 

See [this web accessibility tester](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=959595), where having `#959595` on white fails all tests. We additionally use `#959595` on variants of grey which makes it even more difficult. 

## Expected Behavior

All text in the settings UI should pass basic contrast tests.

## Bug Type

- This bug describes functionality that never worked as expected.

## Steps to Reproduce

1. View the Licensing tab on version 2.5.0 and use the above web accessibility contrast checker to confirm.

## Visuals

![image](https://user-images.githubusercontent.com/930724/59121619-7b6f9080-890d-11e9-9dc7-6f3ba505d2b7.png)

## Possible Solution

I'd suggest all of the current `#959595` to be `#555555` or at minimum `#666666`

## Acceptance Criteria

- [ ] All text on white or grey backgrounds has enough contrast to be read easily without eye strain.
